pythonwhat/Test.py

```diff
@@ -147,73 +98,56 @@ def specific_test(self):
         """
         self.result = self.func(self.obj1, self.obj2)
 
-## Helpers for testing equality
 
-def objs_are(x, y, list_of_classes):
-    return (
-        any([isinstance(x, klass) for klass in list_of_classes]) &
-        any([isinstance(y, klass) for klass in list_of_classes])
-    )
+# Helpers for testing equality
+
+
+def areinstance(x, y, tuple_of_classes):
```

Comment:
U could make it more general:
```python3
def areinstance(*args):
  return all(isinstance(x, args[-1]) for x in args[:-1])
```

Reply:
I thought about that, but didn't do it as it's currently not needed and this has a cleaner signature.